lib/internal/validators.js

```diff
@@ -195,8 +287,17 @@ function validateSignalName(signal, name = 'signal') {
 
     throw new ERR_UNKNOWN_SIGNAL(signal);
   }
+
+  return true;
```

Comment:
```suggestion
```

Reply:
Since jsdoc has `@returns`, it needs to return a boolean, or linter throws error:

```
/Users/yagiz/Developer/node/lib/internal/validators.js
  274:1  warning  JSDoc @returns declaration present but return expression not available in function  jsdoc/require-returns-check
```

Reply:
I'd prefer if we silence the linter rule (or remove the `@returns`) than modifying the actual function.

Reply:
I suppressed it. Thanks for the feedback